runtime/tutor.txt

```diff
@@ -630,7 +630,7 @@ onto other
 lines.
      This sentence is spilling over onto other lines.
 
- Note: J works on all lines in selection, so typing xxx or 3x to
+ Note: J works on all lines in selection, so typing xxxx or 4x to
        select the lines and then a single J will work the same.
```

Comment:
Well...I think this Note does not duplicate the above instructions, because they are two different methods.

One is to **join the next row to the row where the cursor is located**
The other is to **join all lines in selection**

So I think this Note should not need any changes.

And I just found out that `3J` doesn't seem to be working, don't know if anyone else has found that too.

Reply:
Right yes, I'd like to show both methods. As it's currently written, you only see the `4xJ` method and not the 'join the next row to the row where the cursor is located' method.

Reply:
I think

`join the next row to the row where the cursor is located`

This method should already be in tutor.txt inside.

Check tutor.txt line 624

` 2. Type J thrice or 3J to join the sentence to one line.`

Reply:
I think you're looking at the 22.05 version of the tutor - that section got updated in https://github.com/helix-editor/helix/pull/2716/files#diff-0751fcdc9d0b2b023e5c02ee4941114ca1706d04e1ba73c2f4c5ba287c847074R585-R598 which came after 22.05 but hasn't been released yet

Reply:
Oops, you're right😅, I did read it wrong, let's change it to what you suggested.